ixofoundation/ixo-webclient
# feat: implement inactive tabs (#196)

## Scope
https://nonaredteam.atlassian.net/browse/IXO-615

## Work Done
Added and styled inactive tabs 

## Steps to Test

1. See inactive Economy tab for Projects and Cells.
2. Go into cell project and log in with Keysafe, you should see the active Funding tab and inactive Performance tab. 
3. Log out to see inactive Funding tab and inactive Performance tab.

## Screenshots
- Logged in
![Screenshot 2020-06-10 at 12 44 26](https://user-images.githubusercontent.com/35291291/84259043-2c5bcd00-ab18-11ea-9e86-5bc56e8d4f67.png)
